src/tools/kata-ctl/Cargo.toml

```
@@ -12,6 +12,8 @@ edition = "2021"
 [dependencies]
 anyhow = "1.0.31"
 clap = { version = "3.2.20", features = ["derive", "cargo"] }
+reqwest = { version = "0.11", features = ["json","blocking"] }
```

Micro nit: missing space before opening quote:

```suggestion
reqwest = { version = "0.11", features = ["json", "blocking"] }
```

---

thanks @jodh-intel